frontend/services/src/events/eventsService.ts

```diff
@@ -136,6 +146,32 @@ export async function getAllEvents(isActive?: boolean, isPrivate?: boolean) {
   }
 }
 
+export async function getOrganiserEvents(userId: string): Promise<EventData[]> {
+  eventServiceLogger.info(`getOrganiserEvents`);
+  try {
+    const privateDoc = await getDoc(doc(db, "Users", "Active", "Private", userId));
+
+    if (!privateDoc.exists()) {
+      throw new UserNotFoundError(userId); // Or handle accordingly if you need to differentiate between empty and non-existent data
+    }
+    const privateData = privateDoc.data();
+    const organiserEvents = privateData?.organiserEvents || [];
+    const eventDataList: EventData[] = [];
+    for (let i = 0; i < organiserEvents.length; i++) {
```

Comment:
consider using a typescript foreach loop OR a for..of loop. There is no need for the above index 'i' apart from obtaining the event.

Reply:
danny wrote this :<<<<<<<

Reply:
done with for of loop